src/data.rs

```diff
@@ -11,12 +12,19 @@ use libxml::tree::Node;
 use libxml::xpath::Context;
 use libxml::parser::{Parser, XmlParseError};
 
+use rustsenna::sennapath::SENNA_PATH;
+use rustsenna::senna::{Senna, SennaParseOptions};
+use rustsenna::pos::POS;
+use rustsenna::sentence::Sentence as SennaSentence;
```

Comment:
Btw, as a general suggestion, feel free to omit the "rust" prefix in any Rust crate, it's understood and reduces the amount of boilerplate you need to write. This looks perfect:

```rust
use senna::sentence::Sentence
```